api/src/Entity/Period.php

```diff
@@ -115,6 +134,23 @@ public function __construct() {
         $this->materialItems = new ArrayCollection();
     }
 
+    #[ApiProperty(
+        readableLink: true,
+        example: '[{ "dayOffset": 0, "number": 1 }]'
+    )]
+    #[SerializedName('days')]
+    #[Groups('Period:Days')]
+    public function getEmbeddedDays(): Collection {
+        return $this->days;
+    }
+
+    #[ApiProperty(readableLink: true)]
```

Comment:
Bei `getEmbeddedDays` hast du ein Example drin, hier nicht. Müsste man das ergänzen oder fällt er zurück auf das Example der Property?

Reply:
Bei den `getEmbedded*` Methoden brauchts kein Example, das wird sowieso nicht in Swagger angezeigt. Stattdessen werden, wenn man den Return Type korrekt annotiert, die eingebetteten Felder perfekt automatisch angezeigt, inklusive den korrekten Normalisierungs-Gruppen :partying_face: 
```php
    /**
     * @return Day[]
     */
    #[ApiProperty(readableLink: true)]
    #[SerializedName('days')]
    #[Groups('Period:Days')]
    public function getEmbeddedDays(): array {
        return $this->days->getValues();
    }

    #[ApiProperty(readableLink: true)]
    #[SerializedName('camp')]
    #[Groups(['Period:Camp'])]
    public function getEmbeddedCamp(): ?Camp {
        return $this->camp;
    }
```
führt zu folgendem Output in Swagger:
![Screenshot from 2021-08-04 10-13-22](https://user-images.githubusercontent.com/7566995/128146439-a5218641-ebc2-4c83-88e9-88bebee58dfe.png)